src/server/game/Server/WorldSocketMgr.cpp

```diff
@@ -51,6 +51,8 @@ WorldSocketMgr::~WorldSocketMgr()
     delete _instanceAcceptor;
 }
 
+int const boost::asio::socket_base::max_connections;
```

Comment:
is this required to fix something that this PR broke ? because current clang build works fine